### PR TITLE
Fix issue with test suite totals not being reset

### PR DESCRIPTION
**Pull request by Itzalive**
* Fix issue with test suite totals not being reset
* Add unit tests for JUnitBuilder test suite totals

closes issue #32

### CODE REVIEW

File: source/trx2junit/Internal/JUnitBuilder.cs [hunk 1]
```diff
@@ -31,6 +31,12 @@ public void Build()
         //---------------------------------------------------------------------
         private void AddTestSuite(string testSuiteName, IEnumerable<KeyValuePair<Guid, TestDefinition>> tests)
         {
+            _errors = 0;
```
Comment (gfoidl) [inline source/trx2junit/Internal/JUnitBuilder.cs]: Can you move this to a separate method `ResetCounters` (or similar)?

File: tests/trx2junit.Tests/Internal/JUnitBuilderTests/Build.cs [hunk 2]
```diff
@@ -0,0 +1,126 @@
+﻿using NUnit.Framework;
```
Comment (gfoidl) [inline tests/trx2junit.Tests/Internal/JUnitBuilderTests/Build.cs]: Nit: move this namespace below the system-namespaces.

File: tests/trx2junit.Tests/Internal/JUnitBuilderTests/Build.cs [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+using System.Linq;
+using System.Text;
+using System.Xml.Linq;
+
+namespace trx2junit.Tests.Internal.JUnitBuilderTests
+{
+    [TestFixture]
+    public class Build
+    {
+        private Models.Test testData = new Models.Test();
```
Comment (gfoidl) [inline tests/trx2junit.Tests/Internal/JUnitBuilderTests/Build.cs]: Nit: `_testData` (as naming convention used here).

**Review by gfoidl — COMMENTED** [review]
Thanks for submitting this PR!

Some small changes...

File: source/trx2junit/Internal/JUnitBuilder.cs (hunk 1 above)
Comment (Itzalive) [reply source/trx2junit/Internal/JUnitBuilder.cs]: Sure, makes it cleaner

File: tests/trx2junit.Tests/Internal/JUnitBuilderTests/Build.cs (hunk 2 above)
Comment (Itzalive) [reply tests/trx2junit.Tests/Internal/JUnitBuilderTests/Build.cs]: Coding standards are important 😄  changed both of these

**Review by gfoidl — APPROVED** [review]
Thanks!

**Pull request merged**